third_party/libtock-drivers/Cargo.toml

```diff
@@ -0,0 +1,13 @@
+[package]
+name = "libtock_drivers"
+version = "0.1.0"
+authors = ["Tock Project Developers <tock-dev@googlegroups.com>"]
+license = "MIT/Apache-2.0"
```

Comment:
To be valid, the Apache license requires the LICENSE file to be present as well as a copyright+license header on each file. But if it's Tock's code, they have to fix it upstream.

Reply:
> libtock-rs is licensed under either of
> 
> - Apache License, Version 2.0 (LICENSE-APACHE or http://www.apache.org/licenses/LICENSE-2.0)
> - MIT license (LICENSE-MIT or http://opensource.org/licenses/MIT)
> 
> at your option.

If Apache is too complicated, can we pick MIT for this derived work (i.e. the `third_party/libtock-drivers` crate)? Then ultimately this `third_party/libtock-drivers` should hopefully disappear if the relevant API is available in libtock-rs directly.